monitoring/dashboards/KafkaBackendSLOs.json

```diff
@@ -368,6 +368,217 @@
           "show": true
         }
       ]
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "${DS_PROMETHEUS}",
+      "description": "Shows the number of kafka clusters which have all 5 or brokers in state 3 (running as broker).\n\nIn steady state, the line for 5 brokers must be equal to 25. During fault injection, the line for \"4 or 5 brokers\" must be equal to 25.",
```

Comment:
The graph seems normalised, as such the description is not 100% correct. Ok, no -- it's equal to "the number of running kafka clusters"